### PR TITLE
Fixed not load kits.yml file

### PR DESCRIPTION
**Pull request by montlikadani**
I've fixed the error that did not load the kits.yml file again.

### CODE REVIEW

File: Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java [hunk 1]
```diff
@@ -65,6 +65,7 @@ public void convertKits() {
         }
 
         config.save();
+        config.load();
```
Comment (mdcfe) [inline Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java:68]: What does this achieve? The kit won't realistically change between these two method calls.

**Review by mdcfe — CHANGES_REQUESTED** [review]

File: Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java (hunk 1 above)
Comment (montlikadani) [reply Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java:68]: But if the player type _/essentials reload_ command, it does not reload the kits.yml file.

Comment (mdcfe) [reply Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java:68]: This code is not where the configs are reloaded; this is where the one-time migration of kits from `config.yml` to `kits.yml` occurs.

Comment (montlikadani) [reply Essentials/src/com/earth2me/essentials/EssentialsUpgrade.java:68]: Oh... I just did not find the right one.

**Pull request closed without merging**